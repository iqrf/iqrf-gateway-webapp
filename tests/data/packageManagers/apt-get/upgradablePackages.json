[
	{
		"name": "gnome-control-center",
		"oldVersion": "1:3.30.2-5",
		"newVersion": "1:3.30.3-1"
	},
	{
		"name": "gnome-control-center-data",
		"oldVersion": "1:3.30.2-5",
		"newVersion": "1:3.30.3-1"
	},
	{
		"name": "libkf5auth-data",
		"oldVersion": "5.54.0-1",
		"newVersion": "5.54.0-2"
	},
	{
		"name": "libkf5auth5",
		"oldVersion": "5.54.0-1",
		"newVersion": "5.54.0-2"
	},
	{
		"name": "node-lru-cache",
		"oldVersion": "5.1.1-3",
		"newVersion": "5.1.1-4"
	}
]
