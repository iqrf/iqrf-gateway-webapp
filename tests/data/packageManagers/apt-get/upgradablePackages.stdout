Reading package lists...
Building dependency tree...
Reading state information...
Calculating upgrade...
The following packages will be upgraded:
   gnome-control-center (1:3.30.2-5 => 1:3.30.3-1)
   gnome-control-center-data (1:3.30.2-5 => 1:3.30.3-1)
   libkf5auth-data (5.54.0-1 => 5.54.0-2)
   libkf5auth5 (5.54.0-1 => 5.54.0-2)
   node-lru-cache (5.1.1-3 => 5.1.1-4)
5 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Inst gnome-control-center-data [1:3.30.2-5] (1:3.30.3-1 Debian:unstable [all])
Inst gnome-control-center [1:3.30.2-5] (1:3.30.3-1 Debian:unstable [amd64])
Inst libkf5auth5 [5.54.0-1] (5.54.0-2 Debian:unstable [amd64]) []
Inst libkf5auth-data [5.54.0-1] (5.54.0-2 Debian:unstable [all])
Inst node-lru-cache [5.1.1-3] (5.1.1-4 Debian:unstable [all])
Conf gnome-control-center-data (1:3.30.3-1 Debian:unstable [all])
Conf gnome-control-center (1:3.30.3-1 Debian:unstable [amd64])
Conf libkf5auth5 (5.54.0-2 Debian:unstable [amd64])
Conf libkf5auth-data (5.54.0-2 Debian:unstable [all])
Conf node-lru-cache (5.1.1-4 Debian:unstable [all])
